recipes_source/recipes/profiler.py

```diff
@@ -0,0 +1,203 @@
+"""
+PyTorch Profiler
+====================================
+This recipe explains how to use PyTorch profiler and measure time and
+memory consumption of model's operators.
+
```

Comment:
nit: add the - of "the" model's operators.
Sorry realized missing one more

Reply:
thanks)